Core/Core/CourseSync/CourseSyncDownloader/Model/HTMLParser/OfflineFolderPrefix.swift

```diff
@@ -0,0 +1,37 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2024-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import Foundation
+
+public enum OfflineFolderPrefix: String {
+    case page
+    case assignment
+    case announcement
+    case discussion
+    case quiz
+    case calendarEvent
+}
+
+public enum OfflineContainerPrefix: String {
+    case Pages
+    case Assignments
+    case Announcements
+    case Discussions
+    case Quizzes
+    case CalendarEvents
+}
```

Comment:
just a thought: would it make sense to have only one enum with multiple computed properties?
I have no idea for a proper name, but it would ensure no possible mismatches between the currently separate enums.
Feel free to ignore if you disagree.